connmgr/tor.go

```diff
@@ -100,14 +100,11 @@ func TorLookupIP(host, proxy string) ([]net.IP, error) {
 	}
 	if buf[1] != 0 {
 		if int(buf[1]) > len(torStatusErrors) {
```

Comment:
That's what I was thinking. This allows to do `slice[len(slice)]` which will panic.